### ISSUE TITLE
`import/order` should place `../../foo` before `../foo`

### ISSUE THREAD

**Issue by MrHen** [benmosher/eslint-plugin-import#1388]
`import/order` currently understands the difference between `./foo` and `../foo` but it stops caring after that. It would be nice if `../../` was sorted before `../` and `../../../` was before `../../` and so on.

----

Fail:

```js
import w from './foo';
import x from '../foo';
import y from '../../foo';
import z from 'foo';
```

Pass:

```js
import z from 'foo';
import y from '../../foo';
import x from '../foo';
import w from './foo';
```

**Comment by ljharb** [comment]
Is the current behavior the wrong sort order, or is it unordered?

**Comment by MrHen** [comment]
> Is the current behavior the wrong sort order, or is it unordered?

It is unordered.

**Comment by ljharb** [comment]
In that case it makes sense to me to order them, and that'd either be a bugfix or a new option.
